stylo/shape/basic.py

```diff
@@ -99,3 +109,6 @@ def rectangle(x, y):
 class Square(Rectangle):
     def __init__(self, x, y, size):
         super().__init__(x, y, size, size)
+
+    def __repr__(self):
+        return "Sqaure(x={0.x},y={0.y},size={0.width})".format(self)
```

Comment:
hey, just to report there's a typo in this line of code

Reply:
Good spot!